### ISSUE TITLE
🐛 Bug Report: incorrect naming of `GitHub` on git provider integration

### ISSUE THREAD

**Issue by levivannoort** [amplication/amplication#6243]
### What happened?

<img width="1509" alt="image" src="https://github.com/amplication/amplication/assets/73097785/69f56166-b2cb-48a7-9177-44604016b399">
---
<img width="1505" alt="image" src="https://github.com/amplication/amplication/assets/73097785/cdaa6857-ddbd-4188-9983-776102bb72fe">

### What you expected to happen

`Github` should be `GitHub`

### How to reproduce

When creating a new service or at the project level abstraction of adding a git provider the incorrect spelling of `GitHub` can be found.

### Amplication version

1.7.2

**Comment by levivannoort** [comment]
<img width="1509" alt="image" src="https://github.com/amplication/amplication/assets/73097785/29c38baa-a0fc-4cfe-ad3b-ad990066da9d">

When selecting a repository after adding the connection the incorrect naming can also be found.

**Comment by samahrens** [comment]
Hello, can I have this issue? Thanks!

**Comment by yuval-hazaz** [comment]
The relevant code is in 
/Users/yuvalhazaz/projects/amplication.git/amplication/packages/amplication-client/src/Resource/git/GitActions/GitProviderConnection.tsx

It happens because the component is using the enum value as string. 

instead we need to use a mapping from a providerName to DisplayName
something like 

```typeScript
const PROVIDERS_DISPLAY_NAME: {
  [key in models.EnumGitProvider]: string;
} = {
  [models.EnumGitProvider.Github]: "GitHub",
  [models.EnumGitProvider.Bitbucket]: "Bitbucket",
};
```

**Comment by yuval-hazaz** [comment]
I realized that I opened a duplicate issue yesterday 
https://github.com/amplication/amplication/issues/6286

but the duplicate issue was assigned to someone already and he created a PR
https://github.com/amplication/amplication/pull/6287

I suggest to work o this issue on top of the new PR to fix the text in the second location where the problem exists (the PR only fixes one location)

I will approve and merge the first PR. then pull the latest version of the branch "next" and work on top of it to do two things:
1. move the map to a common file
2. use the map in the second location
